proto/cacheclient.proto

```diff
@@ -1025,3 +1028,35 @@ message _SortedSetLengthByScoreResponse {
     _Missing missing = 2;
   }
 }
+
+message _SortedSetUnionStoreRequest {
+  enum AggregateFunction {
+    SUM = 0;
+    MIN = 1;
+    MAX = 2;
+  }
+
+  message _Source {
+    bytes set_name = 1;
+    optional float weight = 2;
+    AggregateFunction aggregate = 3;
```

Comment:
Aggregate function is _across_ sources, not per-source. You need to bump this up a level to the Request itself

Reply:
Yep - I fixed that a minute ago. Thanks for the catch